lib/motors/controller.hpp

```diff
@@ -5,6 +5,7 @@
 
 namespace hyped::motors {
 enum class ControllerStatus { kControllerTemperatureExceeded, kUnrecoverableWarning, kNominal };
+enum class ControllerStates { kInitialisation, kCalibration, kMotorReady, kMotorAccelerating, kMotorBraking, kRunComplete, kEmergencyBreak };
```

Comment:
add a comment to say this is a placeholder until we get actual states from state machine, also I would rename it just because `ControllerStatus` and `ControllerStates` look almost identical and it is messing with my eyes

Reply:
perhaps `MotorStates` @TomLonergan03 ?